sdk/typescript/src/builder/__tests__/Transaction.test.ts

```diff
@@ -11,6 +11,12 @@ it('can construct and serialize an empty tranaction', () => {
 	expect(() => tx.serialize()).not.toThrow();
 });
 
+it('can construct a receiving transaction argument', () => {
```

Comment:
Can we add a test that verifies the effects of the `receiving` helper (i.e. that it's different from the effect of the `object` helper)?

Reply:
I think we can, and I will!

Reply:
Added a test to make sure that the effects generated by these are different.